stockflux-launcher/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "stockflux-search",
+  "name": "stockflux-launcher",
```

Comment:
@jrhenderson1988, can you confirm this change does not break anything? I think it makes sense for stockflux-launcher to have a name "stockflux-launcher".

Reply:
I don't think it will break anything but I can't be certain. I'll try pulling down your branch and building everything to test. Nothing explicitly defines a dependency on the launcher, right? Other than core, components and bitflux, all of the other modules are top level components that are not depended upon by anything else, as far as I know.